hardware/raspberrypi/booteeprom.md

```diff
@@ -92,6 +92,23 @@ Since the release status string is just a subdirectory name then it's possible t
 
 You can change which release stream is to be used during an update by editing the `/etc/default/rpi-eeprom-update` file and changing the `FIRMWARE_RELEASE_STATUS` entry to the appropriate stream.
 
+### EEPROM write protect
+
+Write protecting the EEPROMs on the Raspberry Pi4 Model B requires both a software change and a small board modification. 
+
+**This is only recommended for advanced users or industrial customers.**
+
+By default, neither the bootloader nor the VL805 SPI EEPROMs are write-protected. 
+
+If `eeprom_write_protect=1` is defined in `config.txt` then `recovery.bin` will define the write protect regions such that all of both EEPROMS are write-protected. The write-protect region configuration is then made read-only when the write-protect (`/WP`) pin is pulled low. If `eeprom_write_protect=0` is defined then the write-protect regions are cleared. If `eeprom_write_protect` is not defined then the write-protect bits are not modified.
```

Comment:
Is it _only_ `recovery.bin` that's able to modify the write-protect regions, or is the bootloader itself (once it has been updated) able to modify them too?

Reply:
It's supported by self-update but the bootloader must have already been updated before the property will be recognised. Bootstrapping the update mechanism in the bootloader often confuses new users because they attempt to do everything in one step so for now, it's a separate item